### ISSUE TITLE
dependency problem with ng new

### ISSUE THREAD

**Issue by burhan** [angular/angular-cli#1238]
I am running this on Ubuntu 16.04 LTS; and following the steps from the Google IO 2016 talk, I entered the following command:

```
ng new puppy-love-io --mobile
```
It seems there is an issue with dependencies:

```
Successfully initialized git.
⠼ Installing packages for tooling via npm
├── es6-shim (ambient)
├── node (ambient)
├── angular-protractor (ambient dev)
├── jasmine (ambient dev)
└── selenium-webdriver (ambient dev)

The package @angular/core@2.0.0-rc.3 does not satisfy its siblings' peerDependencies requirements!
Error: The package @angular/core@2.0.0-rc.3 does not satisfy its siblings' peerDependencies requirements!
    at /usr/lib/node_modules/angular-cli/node_modules/angular-cli/node_modules/npm/lib/install.js:125:32
    at /usr/lib/node_modules/angular-cli/node_modules/angular-cli/node_modules/npm/lib/install.js:268:7
    at /usr/lib/node_modules/angular-cli/node_modules/angular-cli/node_modules/npm/node_modules/read-installed/read-installed.js:142:5
    at /usr/lib/node_modules/angular-cli/node_modules/angular-cli/node_modules/npm/node_modules/read-installed/read-installed.js:263:14
    at cb (/usr/lib/node_modules/angular-cli/node_modules/angular-cli/node_modules/npm/node_modules/slide/lib/async-map.js:47:24)
    at /usr/lib/node_modules/angular-cli/node_modules/angular-cli/node_modules/npm/node_modules/read-installed/read-installed.js:263:14
    at cb (/usr/lib/node_modules/angular-cli/node_modules/angular-cli/node_modules/npm/node_modules/slide/lib/async-map.js:47:24)
    at /usr/lib/node_modules/angular-cli/node_modules/angular-cli/node_modules/npm/node_modules/read-installed/read-installed.js:263:14
    at cb (/usr/lib/node_modules/angular-cli/node_modules/angular-cli/node_modules/npm/node_modules/slide/lib/async-map.js:47:24)
    at /usr/lib/node_modules/angular-cli/node_modules/angular-cli/node_modules/npm/node_modules/read-installed/read-installed.js:263:14
```

Pertinent version information:

```
$ node -v && npm -v && ng --version
v6.2.2
3.10.2
Could not start watchman; falling back to NodeWatcher for file system events.
Visit http://ember-cli.com/user-guide/#watchman for more info.
angular-cli: 1.0.0-beta.8
node: 6.2.2
os: linux x64
```

**Comment by dylclem** [comment]
having exact same error

**Comment by mackelito** [comment]
https://github.com/angular/angular-cli/issues/958 same issue?

**Comment by tmguimaraes** [comment]
Doing `npm install` after fixes that. 
But your command should work without need to use `npm install`

**Comment by tmguimaraes** [comment]
same error with Windows 10

**Comment by WimpyKnight** [comment]
Had the same issue and it fixed by:
`npm uninstall -g angular-cli`
`npm install -g angular-cli@latest`

**Comment by mackelito** [comment]
@WimpyKnight are you really running the --mobile flag?

**Comment by tmguimaraes** [comment]
@WimpyKnight tried that, it gave me a `Cannot read property 'makeCurrent' of undefined` when doing ng serve.

**Comment by filipesilva** [comment]
Closed as issue was made obsolete by #1455.
